buddyboss/buddyboss-platform
# Fix #2306 : Fix discussions were not getting in search result

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2306 .

### How to test the changes in this Pull Request:

1. Enable forum search in network search setting ( BuddyBoss--> Settinggs--> Search ) : https://prnt.sc/12ixkyz
2. Try to search any discussion title from any Forum/Sub-Forum from search box in header : https://prnt.sc/12ixnnk
3. Hit Enter and see the result on result page : https://prnt.sc/12ixovv

### Proof Screenshots or Video

https://www.loom.com/share/5194047bc21f4572aedbb1a608760174

### Other information:




### Changelog entry

d08458c2b3effdfc8915134815b1186a1288c074 Improved search query for forums
